src/components/Pannable/index.js

```diff
@@ -0,0 +1,71 @@
+// @flow
+import * as React from 'react'
+import { h, Component } from 'preact'
+import classNames from 'classnames'
+import style from './style.css'
+
+type Props = {
+  children: ?React.Node,
+  className: ?string,
+}
+
+type State = {
+  clientX: number,
+  clientY: number,
+}
+
+export default class Pannable extends Component<Props, State> {
```

Comment:
I'm not sure I understand what's happening in this component. Can you give me some context?

Reply:
Sure @stefaniacardenas .

First of all, my bad this got merged without you seeing it first. 

About this component, we needed to support panning but prevent users from zooming, or using any other gestures that might reload by panning up, or navigate back by panning left.

This in theory can be done with `touch-action: pan-x pan-y`, but this feature is not supported by Safari, and not fully supported by Chrome.

The solution I found was scrolling using touch events, preventing the default behaviour (no pinch-zoom, pan-up, pan-left).